test/Transforms/GlobalOpt/alias-used.ll

```diff
@@ -7,7 +7,7 @@
 @ia = internal alias i8* @i
 
 @llvm.used = appending global [3 x i8*] [i8* bitcast (void ()* @fa to i8*), i8* bitcast (void ()* @f to i8*), i8* @ca], section "llvm.metadata"
-; CHECK-DAG: @llvm.used = appending global [3 x i8*] [i8* bitcast (void ()* @fa to i8*), i8* bitcast (void ()* @f to i8*), i8* @ca], section "llvm.metadata"
```

Comment:
Do we actually run the tests in this directory? If not we should copy the test somewhere that it will get run.

Reply:
I'll create a new PR with a copied test if necessary (based on internal chat).